plugins/plugin-chart-echarts/src/BoxPlot/transformProps.ts

```diff
@@ -105,7 +105,7 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
   else if (xTicksLayout === '90°') axisLabel = { rotate: -90 };
   else if (xTicksLayout === 'flat') axisLabel = { rotate: 0 };
   else if (xTicksLayout === 'staggered') axisLabel = { rotate: -45 };
-  else axisLabel = { show: true };
+  else axisLabel = { rotate: -45 };
```

Comment:
Can we be smart and only do rotation if there are more than x number of x ticks?

Reply:
To really be smart it would have to be something like ticks / width

Reply:
can we just be consistent.? bar chart has it rotate 45 by default.